wakuv2/waku.go

```diff
@@ -302,6 +303,7 @@ func New(nodeKey *ecdsa.PrivateKey, fleet string, cfg *Config, logger *zap.Logge
 		opts = append(opts, node.WithWakuFilterLightNode())
 		waku.defaultShardInfo = shards[0]
 		opts = append(opts, node.WithMaxPeerConnections(cfg.DiscoveryLimit))
+		cfg.EnableStoreConfirmationForMessagesSent = false
```

Comment:
that is not right and needs to be fixed then, because @cammellos is working on top of this to provide this as a setting to user. 
Any idea how to fix it so that user setting prevails or if not defaults are used?

Reply:
Ok, how would default be set? We want for relay to default to true and lightClient to false?

Reply:
@cammellos, please consider fixing this point as well in https://github.com/status-im/status-go/pull/5505 as i had noticed you had pulled changes from this PR. 

I will close this PR since changes are going to get merged from yours anyways.